src/elements/content-sidebar/activity-feed/task-new/AssigneeStatus.js

```diff
@@ -27,8 +27,8 @@ const StatusIcon = ({ status, ...rest }: { status: TaskCollabStatus }) => {
     }
 };
 
-const AssignmentStatus = React.memo<Props>(({ user, status, getAvatarUrl, ...rest }: Props) => (
-    <div className="bcs-task-assignment-status" {...rest}>
+const AssignmentStatus = React.memo<Props>(({ user, status, getAvatarUrl, className, ...rest }: Props) => (
```

Comment:
Looks like `className` is missing from the `Props` type. Should it also have a default value of `''`?